docs/docs-components/PrincipleItem.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+import { Text, Box, Flex, Heading } from 'gestalt';
+import { type Node } from 'react';
+
+type PrincipleItemProps = {|
```

Comment:
nit: this is the only component here, so no need for namespacing, just `Props` will do